docs/modules/capacity_role.rst

```diff
@@ -49,10 +49,6 @@ Parameters
     IP or FQDN of the Unisphere host
 
 
-  universion (False, int, None)
```

Comment:
In All the docs we should not remove the entry as we are just deprecating the param.

Update the doc fragment with deprecation notice and update the rst's automatically with a command.

Even in examples of playbook and module please don't remove it, we will remove the entry when we remove this param from the doc fragment and module permanently.

https://github.com/dell/ansible-powermax/blob/94405189b7c8734546924bca229cbd8102d86e0f/plugins/doc_fragments/powermax.py#L17C4-L19C15

Reply:
@baoy1  That is ok as along that parameter is not being referred/used in the modules it should not impact customers.
If we remove the parameter immediately then all the existing playbooks will get affected. Now we will give 2 years time to modify any existing playbook that has the attribute and remove the parameter later release.